js/run.js

```diff
@@ -1374,4 +1374,8 @@ function mathFunctions(variableName, userInput) {
             return globalEval(variableName + '=' + 'Math.round(' + a + ')')
         }
     }
+    if (userInput.includes('random')) {
+        let variable = parseFloat(globalEval(parametersAsString[1]))
+        return globalEval(variableName + '=' + 'Math.floor(Math.random() *' + variable + ')')
```

Comment:
```suggestion
        return globalEval(variableName + '=' + 'Math.round(Math.random() *' + variable + ')')
```

I believe rounding is a better choice than floor-ing. 🤔